solutions/usaco-574.mdx

```diff
@@ -108,4 +108,38 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+from collections import deque
+
+fullness, que = set(), deque()
```

Comment:
`que`?

Reply:
also what do these variables store? quick comment noting them

Reply:
like queue

Reply:
then just call it `queue`?

Reply:
also, put these 2 variables after reading in the input